dependency-check-maven-config.xml

```diff
@@ -0,0 +1,485 @@
+        version with the overall Camel version - marking as false positive.
+
+        file name: proxy-camel-servlet-2.10.0-SNAPSHOT.jar]]></notes>
+        <cve>CVE-2015-5344</cve>
+    </suppress>
+
+    <suppress>
+        <notes><![CDATA[
+            CVE-2014-1939 is an Android specific issue, and does not apply here
+            file name: google-http-client-1.22.0.jar]]></notes>
+        <cve>CVE-2014-1939</cve>
+    </suppress>
+
+    <suppress>
+        <notes>
+            CVE-2008-0660 is a stack based buffer overflow vulnerability related to ActiveX and
+            several image uploaders. This is unrelated to presto-parser, so marking as a false
+            positive.
+        </notes>
+        <cve>CVE-2008-0660</cve>
+    </suppress>
+
+    <suppress>
+        <notes>
+            CVE-2007-1085 is a cross-site scripting (XSS) vulnerability related to Google Desktop
+            which does not apply here.
+        </notes>
+        <cve>CVE-2007-1085</cve>
+    </suppress>
+
+    <suppress>
+        <notes>
+            CVE-2007-3150 is a JavaScript injection vulnerability related to Google Desktop which
+            does
+            not apply here.
+        </notes>
+        <cve>CVE-2007-3150</cve>
+    </suppress>
+
+    <suppress>
+        <notes>
+            CVE-2010-1807 is related to a client side/browser vulnerability in WebKit. Marking the
+            vulnerability as a false positive since the vulnerable code is not currently used and
+            the code is executed server-side.
+        </notes>
+        <cve>CVE-2010-1807</cve>
+    </suppress>
+
+    <suppress>
+        <notes>
+            CVE-2011-2730 is related to a vulnerability in the VMware SpringSource Spring
+            Framework, where OWASP flags jars that are unrelated or have no dependency on
+            Spring, so marking it as a false positive.
+        </notes>
+        <cve>CVE-2011-2730</cve>
+    </suppress>
+
+    <suppress>
+        <notes>
+            CVE-2011-5034: Applies to
+            org.apache.servicemix.specs.activation-api-1.1-2.5.0.jar/META-INF/maven/org.apache.geronimo.specs/geronimo-activation_1.1_spec/pom.xml
+            ServiceMix embeds some Specs provided by Geronimo but does not use any of the effected
+            libraries.
+        </notes>
+        <cve>CVE-2011-5034</cve>
+    </suppress>
+
+
+    <suppress>
+        <notes><![CDATA[
+                Suppressing vulnerabilities CVE-2013-4221 and CVE-2013-4221 as the offending jar file (org.restlet-2.1.1.jar)
+        is being manually removed from the Solr War and replaced with the fixed version. These should be removed when
+        Solr is updated (DDF-1110). See pom file for details.
+   file name: solr-4.7.2.war: org.restlet-2.1.1.jar
+   ]]></notes>
+        <sha1>E12C23B962C925F2681729AFA1E40066A350AD27</sha1>
+        <cve>CVE-2013-4221</cve>
+    </suppress>
+    <suppress>
+        <notes><![CDATA[
```

Comment:
Looking at this suppression alone doesn't provide any information as to why it's being suppressed.
Should we just copy the notes from 4221?